src/components/Banners/MintWheelsBanner/MintWheelsBanner.tsx

```diff
@@ -0,0 +1,44 @@
+//- React Imports
+import React from 'react';
+
+//- Style Imports
+import styles from './MintWheelsBanner.module.css';
+
+type MintWheelsBannerProps = {
+	title: string;
+	label: string;
+	labelBold: React.ReactNode;
```

Comment:
Make this whole label one ReactNode so we can send in any arbitrary components (countdown timer, etc.)